index.js

```diff
@@ -161,17 +162,11 @@ module.exports = function(source, map) {
 		}
 
 		if (options.emitCss && css.code) {
-			const cssFilepath = compileOptions.filename.replace(
-				/\.[^/.]+$/,
-				`.svelte.css`
-			);
-
+			const resource = posixify(this.resource);
+			const cssPath = `${resource}.${index++}.css`;
```

Comment:
Perhaps `svelte` should be included in the path to make it clear where the CSS is coming from?

Reply:
Oh, I just tried this I realized that `svelte` gets added due to being part of the filename, oops nevermind